DataFormats/SoATemplate/interface/SoALayout.h

```diff
@@ -1682,10 +1731,19 @@ _SWITCH_ON_TYPE(VALUE_TYPE,
         return *this;                                                                                                  \
     }                                                                                                                  \
                                                                                                                        \
+    /* Helper function to convert a ConstView in a View, useful to implement View Blocks accessors */                  \
```

Comment:
Can you remind me why this is needed, and why it is not `private` ?
By default I would reject this.